web/src/main/java/com/bazaarvoice/emodb/web/scanner/config/ScannerConfiguration.java

```diff
@@ -83,6 +83,13 @@
     @JsonProperty ("completeScanRangeQueueName")
     private Optional<String> _completeScanRangeQueueName = Optional.absent();
 
+    // Region for AWS clients such as for SQS and S3.  Typically this is the local region and is introspected from the
+    // EC2 instance, but for debugging purposes it can be explicitly set here.
+    @Valid
+    @NotNull
+    @JsonProperty ("awsRegion")
+    private Optional<String> _awsRegion = Optional.absent();
```

Comment:
This change isn't strictly necessary but without it the unit test runs slowly.  This is because the the call to `Regions.getCurrentRegion()` goes through numerous timeouts and retries.  As a result when running locally on a non-EC2 instance it adds seconds to each of the new unit tests.  With this change the call to `Regions.getCurrentRegion()` is bypassed.